proto/src/determined/project/v1/project.proto

```diff
@@ -10,6 +10,28 @@ import "google/protobuf/timestamp.proto";
 import "google/protobuf/wrappers.proto";
 import "protoc-gen-swagger/options/annotations.proto";
 
+// GeneralColumn is the pre-defined column names for experiment list table.
+enum GeneralColumn {
+  // Unspecified column.
+  GENERAL_COLUMN_UNSPECIFIED = 0;
+  // Column id.
+  GENERAL_COLUMN_ID = 1;
+  // Column name.
+  GENERAL_COLUMN_NAME = 2;
+  // Column discription.
+  GENERAL_COLUMN_DESCRIPTION = 3;
+  // Column tags.
+  GENERAL_COLUMN_TAGS = 4;
+  // Column user.
+  GENERAL_COLUMN_USER = 5;
+  // Column starttime.
+  GENERAL_COLUMN_STARTTIME = 6;
+  // Column duration.
+  GENERAL_COLUMN_DURATION = 7;
+  // Column state.
+  GENERAL_COLUMN_STATE = 8;
+}
```

Comment:
**question:** There might be a couple of general options missing. Lookin at the UI:
- Resource Pool
- Progress
- Forked
- Searcher Type
- Trial Count
- Checkpoint Size
- Checkpoint Count
<img width="450" alt="Screenshot 2023-03-22 at 9 36 05 AM" src="https://user-images.githubusercontent.com/220971/226956992-63025259-d2c5-4c16-91cd-a2cd42a48fe2.png">

Reply:
I've only included the ones presented in the figma, I thought eliminating some columns was also part of design, but I can definitely add them back.